src/sql/workbench/api/common/extHostQueryEditor.ts

```diff
@@ -58,11 +58,11 @@ export class ExtHostQueryEditor implements ExtHostQueryEditorShape {
 		this._nextListenerHandle++;
 	}
 
-	public $onQueryEvent(handle: number, fileUri: string, event: IQueryEvent): void {
+	public $onQueryEvent(handle: number, fileUri: string, providerId: string, event: IQueryEvent): void {
```

Comment:
probably put the `providerId` as 1st or 2nd parameter

Reply:
done